pypowsybl/network.py

```diff
@@ -94,6 +100,15 @@ def __setstate__(self, state):
         n = _pypowsybl.load_network_from_string('tmp.xiidm', xml, {})
         self._handle = n
 
+    def activate_per_unit(self):
+        self.per_unit = True
+
+    def desactivate_per_unit(self):
```

Comment:
With the separate "view" approaches, no need anymore to disable per-unit or change the nominal S:
you just use the base network or ask for a view per-united with another nominal S.